WTWSMS/common/dynasties/00_dynasties.txt

```diff
@@ -32444,13 +32444,38 @@
 	culture = "italian"
 	religion = "catholic"
 }
+1051154 = {
+	name = "Solomonid"
+	culture = "greek"
+	religion = "nicene"
+}
```

Comment:
Previously we decided against using "id" for Greek characters to instead use "ites", as you have done for the other dynasty names.

Reply:
Oh yes! I forgot about it. I'll change it ASAP.